parlai/tasks/blended_skill_talk/worlds.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
```

Comment:
looks fine.
when https://github.com/facebookresearch/ParlAI/pull/2497 lands, we can inherit from the base class and get rid of some of this

Reply:
Yes - subclassed from `InteractiveWorld`